object_manager_test.go

```diff
@@ -521,4 +523,40 @@ var _ = Describe("Object Manager", func() {
 			Expect(err).To(BeNil())
 		})
 	})
+
+	Describe("Get IPAddressInfo", func() {
+		cmpType := "Docker"
+		tenantID := "01234567890abcdef01234567890abcdef"
+		sf := map[string]string{
+			"ip_address": "dummy",
+		}
+		zdFakeConnector := &fakeConnector{
+			getObjectQueryParams: NewQueryParams(false, sf),
+			getObjectObj:         &IPv4Address{},
+			resultObject: []IPv4Address{{
+				Ref:         "ipv4address/Li5pcHY0X2FkZHJlc3MkMTAuMTk3LjM4LjIzNS8w:dummy",
+				IpAddress:   "dummy",
```

Comment:
Please return value of ip address to some value instead of `dummy`.

Reply:
Addressed.